run.py

```diff
@@ -1,59 +1,48 @@
 from flask import Flask, jsonify, abort, request
 
-app =  Flask(__name__)
-
-orders = [
-    {
-        'orderId': 1,
-        'orderTitle': 'Ghee',
-        'orderDescription': 'Just what i want'
-    },
-    {
-        'orderId': 2,
-        'orderTitle': 'Corn',
-        'orderDescription': 'I love corn too'
-    },
-    {
-        'orderId': 3,
-        'orderTitle': 'Milk',
-        'orderDescription': 'I took lots of milk'
-    }
-]
-#home page of the api
+app = Flask(__name__)
+
+orders = []
+# home page of the api
 
 
 @app.route('/', methods=['GET'])
 def index():
-    return jsonify({'Welcome':"Hello"})
+    return jsonify({'Welcome': "Hello"})
 
-#displaying all the orders in the list
+# displaying all the orders in the list
 
 
 @app.route('/api/v1/orders', methods=['GET'])
 def getOrders():
-    return jsonify({'orders': orders})
```

Comment:
i think this return is irrelevant

Reply:
ooh I have replaced this with a resource version